pennylane_qiskit/remote.py

```diff
@@ -25,16 +25,95 @@ class RemoteDevice(QiskitDevice):
+            to estimate expectation values and variances of observables.
+        session (Session): a Qiskit Session to use for device execution. If none is provided, a session will
+            be created at each device execution.
+        compile_backend (Union[Backend, None]): the backend to be used for compiling the circuit that will be
+            sent to the backend device, to be set if the backend desired for compliation differs from the
+            backend used for execution. Defaults to ``None``, which means the primary backend will be used.
+        **kwargs: transpilation and runtime keyword arguments to be used for measurements with Primitives.
+            If an `options` dictionary is defined amongst the kwargs, and there are settings that overlap
+            with those in kwargs, the settings in `options` will take precedence over kwargs. Keyword
+            arguments accepted by both the transpiler and at runtime (e.g. ``optimization_level``)
+            will be passed to the transpiler rather than to the Primitive.
+
+    **Example:**
+
+    .. code-block:: python
+
+        import pennylane as qml
+        from qiskit_ibm_runtime import QiskitRuntimeService
+
+        service = QiskitRuntimeService(channel="ibm_quantum")
+        backend = service.least_busy(n_qubits=127, simulator=False, operational=True)
+        dev = qml.device("qiskit.remote", wires=127, backend=backend)
+
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.RX(x, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(1))
+
+    >>> circuit(np.pi/3, shots=1024)
+    0.529296875
+
+    This device also supports the use of local simulators such as ``AerSimulator`` or fake backends such as ``FakeManila``.
+
+    .. code-block:: python
+
+        import pennylane as qml
+        from qiskit_aer import AerSimulator
+
+        backend = AerSimulator()
+        dev = qml.device("qiskit.remote", wires=5, backend=backend)
+
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.RX(x, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(1))
+
+    >>> circuit(np.pi/3, shots=1024)
+    0.49755859375
+
+    We can also change the number of shots, either when initializing the device or when we execute
+    the circuit. Note that the shots number specified on circuit execution will override whatever
+    was set on device initialization.
+
+    .. code-block:: python
+
+        dev = qml.device("qiskit.remote", wires=5, backend=backend, shots=4000)
+
+        @qml.qnode(dev)
+        def circuit(x):
+            qml.RX(x, wires=[0])
+            qml.CNOT(wires=[0, 1])
+            return qml.expval(qml.PauliZ(1))
+
+        circuit(np.pi/3) # this will run with 4000 shots
+        circuit(np.pi/3, shots=10000) # this will run with 10000 shots
+        circuit(np.pi/3) # this will run with 4000 shots
```

Comment:
Maybe use the `sample` measurement and show the length of the output to confirm with comments.